Cargo.lock

```diff
@@ -1402,12 +1531,23 @@ checksum = "b9e0384b61958566e926dc50660321d12159025e767c18e043daf26b70104c39"
 
 [[package]]
 name = "idna"
-version = "0.5.0"
+version = "1.0.3"
```

Comment:
This resolves https://github.com/advisories/GHSA-h97m-ww89-6jmq.

Reply:
I ran `cargo update url` which made all the changes to `Cargo.lock` itself.

Reply:
Huh. I swear I tried that on Lernie to solve the same problem and it didn't work. Let me give that a shot again.

Yep, it worked. Nice!